server/lib/request-handler.js

```diff
@@ -13,17 +74,17 @@ exports.termSearch = function(req, res) {
   //req.body.term comes in as an array of search terms input form user
   //convert search term array to string
   var searchTerms = req.body['term'].slice();
-  searchTerms.toString();
-  searchTerm.concat(searchTerms);
+  searchTerms = searchTerms.toString();
+  searchTerm = searchTerm.concat(searchTerms);
 
   //sends post request to Twinword API with user entered search term
   unirest.post("https://twinword-word-associations-v1.p.mashape.com/associations/")
-    .header("X-Mashape-Key", wordAssocAPIk['key'])
+    .header("X-Mashape-Key", apiKey.wordAssocAPIk['key'])
     .header("Content-Type", "application/x-www-form-urlencoded")
     .header("Accept", "application/json")
     .send(searchTerm)
     .end(function (result) {
       //sends word association back to client as an array of words
-      res.writeHead(200).send(result.body['associations_array']);
+      res.send(result.body['associations_array']);
   });
```

Comment:
Need to handle error conditions from Twinwords API and forward some sort of relevant error message to the front end.